### DISCUSSION TITLE
Decrypt SSL Traffic

### DISCUSSION THREAD

**Discussion by samtheruby** [Security-Onion-Solutions/securityonion#11394] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

20

### RAM

64GB

### Storage for /

200GB

### Storage for /nsm

12TB

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hey all,
I'm trying to figure out if there is any way to decrypt ssl traffic and what the easiest way would be. I mainly want strelka to be able to scan all executables on the network, but as most of them are encrypted, it wont allow me to scan them. People have mentioned SSLsplit but I have not been able to get that to work. Is there any guides or ways of getting this working? Thanks in advance!

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
Most folks use their proxy or NGFW to decrypt SSL and then send the decrypted traffic to Security Onion. 

Depending on your requirements, another option might be Polar Proxy:
https://www.netresec.com/?page=PolarProxy
